packages/client/components/NewMeetingHowToSteps.tsx

```diff
@@ -25,19 +25,27 @@ const STEPS = {
     'Invite your team to vote on effort/value',
     'Discuss differences in scores',
     'Export final score to your tracker'
+  ],
+  teamPrompt: [
+    'Open the board and share it with your team',
+    'Share what you’re working on',
```

Comment:
+1 having the verb be "share" for both of these feels weird.
Feel free to wordsmith a little

CC @ackernaut

Reply:
@mattkrick I've created a separate issue for it: https://github.com/ParabolInc/parabol/issues/6224 so we don't miss it